### ISSUE TITLE
Novel view backnorm

### ISSUE THREAD

**Issue by fuyabo** [caiyuanhao1998/X-Gaussian#13]
Great work! Just smashed that star button! When you synthesize new views from the 5 views, I wonder how you did the backnorm (scale back to the correct intensity) in order for the traditional methods to work? It seems the synthesized images are all normalized, e.g. in the code, image = torch.clamp(renderFunc(viewpoint, scene.gaussians, *renderArgs)["render"], 0.0, 1.0) image_backnorm = (viewpoint.max_value - viewpoint.min_value) * image + viewpoint.min_value.   We do not know viewpoint.max_value/min_value since they come from ground truth and is unknown.  I appreciate your hard work!

**Comment by caiyuanhao1998** [comment]
Hi, thanks for your interest!

Please use the average min and max values of the five given views to backnorm. Directly querying the CT volume from the Gaussian point clouds is very difficult. If you want to directly reconstruct CT slides, I suggest you refer to our CVPR 2024 work - SAX-NeRF:

https://github.com/caiyuanhao1998/SAX-NeRF

It is based on NeRF and can jointly reconstruct novel views and CT volume.

**Comment by fuyabo** [comment]
I experimented quite a lot and noticed that by reducing the number of training views from 50 to 5, (equal angular spaced), the quality of the synthesized novel view degraded significantly, though the synthesized view for the training angle is always almost perfect. It looks like the learnt 3D gaussian overfits to the training images, where the splatted 2D view is very similar to the training images. However, for new angles, it collapses, generating a lot of hallucinations. Would you please point to me what I might have done incorrectly, and any tips to get as good results as your paper indicated that from only 5 views, you can get excellent CT when reconstructed using newly synthesized projections. Really appreciate your response!
